targets/esp32/jshardware.c

```diff
@@ -354,8 +341,9 @@ void jshSetOutputValue(JshPinFunction func, int value) {
   }
   else{
     pin = ((func >> JSH_SHIFT_INFO) << 4) + ((func >> JSH_SHIFT_TYPE) & 15);
-    value >> (16 - PWMTimerBit);
-    setPWM(pin,value);
+    // Was warning: statement with no effect [-Wunused-value] - changed to value - needs testing!
```

Comment:
The value for jshSetOutputValue is given as 16 bit.
DAC is set to handle 10 bit.
Therefore this line should convert the 16 bit value to a 10 bit value.
Oh, should the line be: value = value >> (16 - PWMTimerBit); ?